frontend/src/components/Robots/RobotAvatar/index.tsx

```diff
@@ -29,45 +40,53 @@ const RobotAvatar: React.FC<DepthChartProps> = ({ order }) => {
     </div>
   );
 
-  const statusBadgeColor = () => {
-    if (!order) {
-      return;
-    }
-    if (order.maker_status === 'Active') {
-      return 'success';
-    }
-    if (order.maker_status === 'Seen recently') {
-      return 'warning';
-    }
-    if (order.maker_status === 'Inactive') {
-      return 'error';
+  const avatarWithBadges = () => {
+    let component = (
+      <Avatar
+        className={avatarClass}
+        alt={nickname}
+        src={avatarSrc}
+        imgProps={{
+          onLoad,
+        }}
+      />
+    )
+
+    if (statusColor) {
+      component = (
+        <Badge variant='dot' overlap='circular' badgeContent='' color={statusColor}>
+          {component}
+        </Badge>
+      )
     }
-  };
 
-  return order ? (
-    <Tooltip placement='right' enterTouchDelay={0} title={t(order.maker_status) || ''}>
-      <Badge variant='dot' overlap='circular' badgeContent='' color={statusBadgeColor()}>
+    if (orderType !== undefined) {
+      component = (
         <Badge
           overlap='circular'
           anchorOrigin={{ horizontal: 'right', vertical: 'bottom' }}
           badgeContent={statusBadge}
         >
-          <div style={{ width: 45, height: 45 }}>
-            <SmoothImage
```

Comment:
I wonder whether we can keep using `SmoothImage`, at least for the Order Book avatars. It looks a bit ugly when so many images are loading slowly from top to bottom. With `SmoothImage` they fade-in after fully loaded.
![bookloads](https://user-images.githubusercontent.com/90936742/191733125-98c95446-f167-49eb-ac54-e652a4712f3c.png)

Reply:
Let me try to find out if we can do something with the avatar component